.github/workflows/release.yml

```diff
@@ -5,30 +5,47 @@ on:
     types:
       - created
 
+env:
+  AWS_S3_BUCKET: ksp-ckan
+
 jobs:
-  release:
-    runs-on: ubuntu-latest
+  test-release:
+    uses: ./.github/workflows/test.yml
 
-    container:
-      image: mono:latest
+  smoke-inflator:
+    uses: ./.github/workflows/smoke.yml
 
+  sign-release:
+    runs-on: ubuntu-latest
+    needs:
+      - test-release
+      - smoke-inflator
+    outputs:
+      artifact-url: ${{steps.sign.outputs.signing-request-id }}
```

Comment:
I'm trying to figure out where this `artifact-url` output is used. Any clues?

Reply:
It will be used when we want to download it when we have the real certificate available.

Reply:
Cool, I figured it would be something like that, but I wanted to make sure I wasn't missing some nuance in what's already there. Thanks.